### PR TITLE
feat(docs): style the blog home

### PR DESCRIPTION
**Pull request by sstraatemans**
creating the home page for Blog.
The story is not done. but the PR was getting to big.

Things that need to be done to complete the story:
- the first blog post, should have a bigger title and avatar
- avatars should have different background colors
- sidemenu is empty now

In the designs there is a header image. we dont have that data yet.

### CODE REVIEW

**Review by realdreamer — APPROVED** [review]
LGTM 👍🏽

File: packages/apps/docs/src/components/Blog/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import React, { FC } from 'react';
+
+interface IProps {
+  name?: string;
+}
+
+export const Avatar: FC<IProps> = ({ name }) => {
+  if (!name) return null;
+
+  const getInitials = (name: string): string => {
```
Comment (barthuijgen) [inline packages/apps/docs/src/components/Blog/Avatar/Avatar.tsx:12]: This method could be moved outside of the react component, also it seems a bit naive in handling middle names. Are you sure the implementation is desired as is?

File: packages/apps/docs/src/components/InfiniteScroll/InfiniteScroll.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+export const InfiniteScroll: FC<IProps> = ({
+  handleLoad,
+  handleRetry,
+  isLoading,
+  error,
+  isDone,
+}) => {
+  const loadRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
```
Comment (barthuijgen) [inline packages/apps/docs/src/components/InfiniteScroll/InfiniteScroll.tsx:23]: can look into returning a cleanup function in the `useEffect` to clean up the `IntersectionObserver`

File: packages/apps/docs/src/pages/api/blog.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+): Promise<void> => {
+  const { limit = 10, offset = 0 } = req.query as unknown as {
+    limit: number;
+    offset: number;
+  };
+
+  const data = getInitBlogPosts(menuData as IMenuData[], offset, limit);
+
+  res.status(200).json(data);
+  res.end();
```
Comment (barthuijgen) [inline packages/apps/docs/src/pages/api/blog.ts:18]: I'm pretty sure just `res.json()` inherently does `.status(200)` and also acts as `.end()` so those are both redundant.

Comment (sstraatemans) [reply packages/apps/docs/src/pages/api/blog.ts:18]: cool. lessons learned

File: packages/apps/docs/src/components/InfiniteScroll/InfiniteScroll.tsx (hunk 2 above)
Comment (sstraatemans) [reply packages/apps/docs/src/components/InfiniteScroll/InfiniteScroll.tsx:23]: done

File: packages/apps/docs/src/hooks/useGetBlogs.ts [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+  useEffect(() => {
+    if (!isLoading || error !== undefined) return;
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    get();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isLoading]);
+
+  useEffect(() => {
+    console.log(222);
```
[the remark this answers is not in the document]
Comment (sstraatemans) [reply packages/apps/docs/src/hooks/useGetBlogs.ts:61]: fack

**Review by barthuijgen — APPROVED** [review]
LGTM :)

**Pull request merged**